docs/admin/machine.rst

```diff
@@ -303,6 +303,27 @@ To enable this service, add ``weblate.machinery.youdao.YoudaoTranslation`` to
     :setting:`MT_YOUDAO_SECRET`
     `Youdao Zhiyun Natural Language Translation Service <https://ai.youdao.com/product-fanyi.s>`_
 
+.. _netease-translate:
+
+Netease Sight API machine translation
+-------------------------------------
+
+.. versionadded:: 3.2
+
+Machine translation service provided by Netease.
+
+This service uses an API and you need to obtain key and secret from Netease.
+
+To enable this service, add ``weblate.machinery.youdao.NeteaseSightTranslation`` to
+:setting:`MT_SERVICES` and set :setting:`MT_NETEASE_KEY` and
+:setting:`MT_NETEASE_SECRET`.
+
+.. seealso::
+
+    :setting:`MT_NETEASE_KEY`,
+    :setting:`MT_NETEASE_SECRET`
```

Comment:
The documentation for setting is missing, also see [build failure](https://travis-ci.org/WeblateOrg/weblate/jobs/437961012#L934).